src/App.php

```diff
@@ -1158,4 +1169,31 @@ private function getRenderedPortals(): array
 
         return $portals;
     }
+
+    protected function emitResponse(): void
+    {
+        $http_line = sprintf('HTTP/%s %s %s',
+            $this->response->getProtocolVersion(),
+            $this->response->getStatusCode(),
+            $this->response->getReasonPhrase()
+        );
+        header($http_line, true, $this->response->getStatusCode());
```

Comment:
```suggestion
        http_response_code($this->response->getStatusCode());
```
I think `header` will not set the internal php status code and read by `http_response_code()` will not be possible otherwise (in theory, I know, we are about to exit anyway...)

Reply:
OK